src/code/modules/notifications.js

```diff
@@ -1,26 +1,58 @@
 import Immutable from 'seamless-immutable';
 import { GAMETES_RESET } from '../modules/gametes';
+import actionTypes from '../action-types';
+import t from '../utilities/translate';
 
 export const GUIDE_CONNECTED = "Guide connected";
 export const GUIDE_MESSAGE_RECEIVED = "Guide message received";
 export const GUIDE_ALERT_RECEIVED = "Guide alert received";
 export const GUIDE_ERRORED = "Guide errored";
+export const ADVANCE_NOTIFICATIONS = "Advance notifications";
+export const CLOSE_NOTIFICATIONS = "Close notifications";
 
-const initialState = Immutable([]);
+const initialState = {
+  messages: Immutable([]),
+  closeButton: null
+};
 
 export default function notifications(state = initialState, action) {
   switch (action.type) {
     case GUIDE_MESSAGE_RECEIVED:
       if (action.data) {
-        return state.concat(action.data.message.asString());
+        let currentMessage = state.messages.length > 0 ? state.messages[0] + " " : "";
+        return Object.assign({}, state, {messages: [currentMessage + action.data.message.asString()]});
       }
       else
         return state;
+    case actionTypes.MODAL_DIALOG_SHOWN:
```

Comment:
This is a little bit of a hack - old modal dialog messages are now handled here for display. Really, the actions should be renamed for clarity, since we no longer show a modal dialog at all